resources/extra-large-conference/prosody-v@.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 ### see man systemd.unit
-Description=Prosody vX (visitor vX) JVB XMPP Server
+Description=Prosody v%i (visitor v%i) JVB XMPP Server
```

Comment:
BTW, maybe the change can be ported to the upstream Prosody to support templates out of the box?
I'll ask them in their chat